contacts/contacts/Contact.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Contact.swift
+//  contacts
+//
+//  Created by Evelina on 24.10.2020.
+//  Copyright © 2020 Evelina. All rights reserved.
+//
+
+import UIKit
+
+class Contact{
+    
+    public let name:String?
+    public let surname:String?
+    public let telephoneNumber:String?
+    
+    init(name:String?,surname:String?,telephoneNumber:String?) {
```

Comment:
Нужно больше пробелов.
```suggestion
    init(name: String?, surname: String?, telephoneNumber: String?) {
```